data/avro-schema/src/main/resources/avro/net/corda/data/p2p/LinkOutHeader.avsc

```diff
@@ -3,6 +3,11 @@
   "name": "LinkOutHeader",
   "namespace": "net.corda.p2p",
   "fields": [
+    {
+      "doc": "The source identity (x500 name + group ID)",
```

Comment:
Slightly nit picky: Can we order this `destination` first then `source` to be the same as `AuthenticatedMessageHeader`.

Reply:
sure